README.md

```diff
@@ -6,7 +6,12 @@ To test your server against the Cob Spec suite of tests, follow the instructions
 
     git clone git@github.com:8thlight/cob_spec.git
     cd cob_spec
-    mvn package
+    mvn package  
+  
+Setup
+------------
+
+Cob Spec requires maven and JDK 8 to be installed.  
```

Comment:
It might be better if these requirements were stated before the instructions.

"maven" should be "Maven" when referring to it by name.

Finally, it might be helpful to make "Maven" and "JDK 8" link to their respective project pages or installation instructions pages.